jobbergate-cli/jobbergate_cli/subapps/job_submissions/app.py

```diff
@@ -50,12 +52,26 @@ def create(
     ),
     execution_directory: Optional[Path] = typer.Option(
         None,
-        help="""
+        help=dedent(
+            """
             The path on the cluster where the job script should be executed.
             If provided as a relative path, it will be converted as an absolute path from your current
-            working directory. If you use "~" to denote your home directory, the path will be expanded to an absolute
-            path for your home directory on *this* machine.
-        """,
+            working directory. If you use "~" to denote your home directory, the path will be expanded to an
+            absolute path for your home directory on *this* machine.
+            """
+        ).strip(),
+    ),
+    execution_parameters: Optional[Path] = typer.Option(
```

Comment:
I'm not so sure about this requiring a JSON file path. Why did you decide to go this route?

Reply:
Oh, I see. Ok.